lib/xenops_utils.ml

```diff
@@ -290,7 +290,7 @@ module Unixext = struct
   let write_string_to_file fname s =
     atomic_write_to_file fname 0o644 (fun fd ->
         let len = String.length s in
-        let written = Unix.write fd s 0 len in
+        let written = Unix.write fd (Bytes.unsafe_of_string s) 0 len in
```

Comment:
Is this safe? `s` is passed as a parameter and thus can be shared.

Reply:
This is used to write the result of `Jsonrpc.to_string`, I think it's better to avoid the copy in this case